### DISCUSSION TITLE
What if Prism could configure servers?

### DISCUSSION THREAD

**Discussion by benjamin051000** [PrismLauncher/PrismLauncher#1070] [Ideas]
Being able to quickly download/install mods to a client is the primary goal of Prism.

Are there any projects that do the same for servers? If not, would Prism be a good candidate to perform this? I would love an easy way to:
- Download a server (paper/forge/vanilla/etc.)
- Install mods/plugins to the server
- Do other server config stuff
- Run it

Since this is essentially the same flow as client-side modding, I could see this being added to Prism. Let me know what you all think.

**Comment by ghost** [comment]
If it's only LAN servers then it's doable

**Reply by benjamin051000** [reply]
Right, but port forwarding/UPnP/etc. is technically not related to creating the server. 

You actually likely could use a UPnP port mapper to automate forwarding a particular port, but that's probably out of scope for this idea. 

Setting up the server itself, just like setting up the MC Client, is probably all we'd want to do here.

**Reply by benjamin051000** [reply]
I wonder if https://e4mc.link/ would make this easier...

**Comment by benjamin051000** [comment]
See #864

**Reply by TheKodeToad** [reply]
in fact - I think the issue tracker is a better place so i'll close this :P
